### ISSUE TITLE
Backspace broken in Facebook status update input box.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [noctarius/chromebackspaceonlinux#9]
```
What steps will reproduce the problem?
1. Try to backspace in the status input box.

What is the expected output? What do you see instead?
Chrome moves backwards in history, when a character should be erased 
instead.

Please provide any additional information below.
I disabled the extension for facebook.com because I was hitting this bug so 
much.  Now backspace still doesn't work in the textbox (I assume the 
extension is suppressing the event), but at least I don't keep losing my 
updates.
```

Original issue reported on code.google.com by `mrlacha...@gmail.com` on 29 Mar 2010 at 1:21

**Comment by GoogleCodeExporter** [comment]
```
They used a div layer with some self done javascripts to make it working as a 
textfield.

Fixed in 0.45
```

Original comment by `noctariushtc@googlemail.com` on 29 Mar 2010 at 10:11
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This is also broken on pages that are not the homepage.  The check
target.baseURI == 'http://www.facebook.com/'
should probably be a substring instead.
```

Original comment by `mrlacha...@gmail.com` on 29 Mar 2010 at 10:57
